packages/botonic-plugin-flow-builder/src/action/index.tsx

```diff
@@ -83,6 +84,10 @@ async function getContents(
     return await getContentsByFirstInteraction(context)
   }
 
+  if (request.input.type === INPUT.EVENT_INITIAL_QUEUE_POSITION) {
+    // TODO: Think how we will get this content
+  }
```

Comment:
are we not gonna implement this now?

Reply:
yes, I'm leaving this after I can try with the whole integration backend/fbuilder app and this